CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Removed
```

Comment:
CHANGELOG is automatically generated from CI, so there is no need for manual change.